app/models/text.rb

```diff
@@ -13,5 +13,6 @@ class Text < ApplicationRecord
     rails: 4,
     php: 5
   }
-
+  
+  RAILS_GENRE_LIST = %w[basic git ruby rails]
```

Comment:
配列の定数は `freeze` で変更できないようにしておきましょう。

```suggestion
  RAILS_GENRE_LIST = %w[basic git ruby rails].freeze
```